src/test/BiscuitDisplayerTest.php

```diff
@@ -4,9 +4,29 @@
 
 require_once '../vendor/autoload.php';
 
+use BiscuitApp\Biscuit;
+use BiscuitApp\BiscuitDisplayer;
 use PHPUnit\Framework\TestCase;
 
 class BiscuitDisplayerTest extends TestCase
 {
+    public function testSuccessBiscuitDisplayer()
+    {
+        $biscuitMock = $this->createMock(Biscuit::class);
+        $biscuitMock->expects($this->once())->method('getName')->willReturn('Bourbon');
+        $biscuitMock->expects($this->once())->method('getImg')->willReturn('assets/images/bourbons-unedited.jpg');
+        $biscuitMock->expects($this->once())->method('getRDT')->willReturn('7');
+        $input = [$biscuitMock];
+
+
+        $expectedOutput = '<div class="card"><div class="item-header"><h2>Bourbon</h2></div><div class="item-img"><img src="assets/images/bourbons-unedited.jpg" alt="Bourbon biscuit"></div><div class="item-data"><h3>RDT: 7</h3></div></div>';
```

Comment:
i would like you to create multiple mock biscuits (maybe 2) and add both to the array, just to make sure that it handles multiple biscuits in the correct way. so create another biscuit, add it to the array and adjust the expected output accordingly